app/human-detector-app/screens/LoginScreen.tsx

```diff
@@ -15,10 +16,22 @@ interface Props {
  *
  * @returns LoginScreen component
  */
-export default function LoginScreen({ onTokenResponse }: Props): React.ReactElement {
+export default function LoginScreen({ onSuccessfulLogin }: Props): React.ReactElement<Props> {
   return (
     <View>
-      <KeyCloakButton onTokenResponse={onTokenResponse} />
+      <Button
+        title="Login"
+        onPress={() => {
+          const clientId = Constants.manifest?.extra?.clientId;
+          const issuerOrigin = Constants.manifest?.extra?.keycloakUrl;
+          const issuer = new URL('/realms/users', issuerOrigin);
+          const redirectUri = makeRedirectUri();
```

Comment:
Is fixing redirectUri for the managed workflow gonna be something in a different PR?

Reply:
Yeah I think I'll add that in a separate PR if that's OK, this one is pretty big as-is